sru-scripts/manual/ec2-sru

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+
+set -ex
+
+SRU_VARS=./sru-scripts/sru-vars.template
+
+if [ ! -f $SRU_VARS ]; then
+   echo "Missing config environment file $SRU_VARS."
+   exit 1
+fi
+
+. $SRU_VARS
+
+parse_args $0 $@
+assert_expected_vars PROPOSED_SCRIPT LP_USER SAMPLE_CLOUDCONFIG SSHOPTS \
+    PRESERVE_INSTANCE
```

Comment:
Since `PRESERVE_INSTANCE` is a flag variable, I would not expect it to be present all the times we run the script. For example, if we want to destroy the instance after running the test, this variable should not be present

Reply:
no worries about that, it's really just documentation of the 'sru-vars' used by the script in this case. PRESERVE_INSTANCE=false by default in sru-vars.template, It only get set to true if --preserve-instance is provided as an argument by `parse_args()`.  Since PRESERVE_INSTANCE will never by "UNSET" on not present,  `assert_expected_vars()` will not error in either case.